[2025-01-31T05:05:58.220+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag1_v3.second_task scheduled__2022-03-26T00:00:00+00:00 [queued]>
[2025-01-31T05:05:58.242+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag1_v3.second_task scheduled__2022-03-26T00:00:00+00:00 [queued]>
[2025-01-31T05:05:58.243+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2025-01-31T05:05:58.285+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): second_task> on 2022-03-26 00:00:00+00:00
[2025-01-31T05:05:58.298+0000] {standard_task_runner.py:57} INFO - Started process 12134 to run task
[2025-01-31T05:05:58.306+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag1_v3', 'second_task', 'scheduled__2022-03-26T00:00:00+00:00', '--job-id', '3683', '--raw', '--subdir', 'DAGS_FOLDER/dag_example.py', '--cfg-path', '/tmp/tmpbit6vb5h']
[2025-01-31T05:05:58.308+0000] {standard_task_runner.py:85} INFO - Job 3683: Subtask second_task
[2025-01-31T05:05:58.451+0000] {task_command.py:415} INFO - Running <TaskInstance: dag1_v3.second_task scheduled__2022-03-26T00:00:00+00:00 [running]> on host f8bdfcb8bca5
[2025-01-31T05:05:58.672+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='anhtupham99' AIRFLOW_CTX_DAG_ID='dag1_v3' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2022-03-26T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-03-26T00:00:00+00:00'
[2025-01-31T05:05:58.674+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-01-31T05:05:58.678+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo I am the second task, I run after the first task']
[2025-01-31T05:05:58.704+0000] {subprocess.py:86} INFO - Output:
[2025-01-31T05:05:58.708+0000] {subprocess.py:93} INFO - I am the second task, I run after the first task
[2025-01-31T05:05:58.708+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-01-31T05:05:58.770+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag1_v3, task_id=second_task, execution_date=20220326T000000, start_date=20250131T050558, end_date=20250131T050558
[2025-01-31T05:05:58.818+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-01-31T05:05:58.854+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
